Error compiling mafExtrator

I received this error while making mafExtrator

```mkdir -p ../bin/
gcc -std=c99 -Wno-unused-but-set-variable -O3 -Wall -Werror --pedantic -funroll-loops -DNDEBUG -Wshadow -Wpointer-arith -Wstrict-prototypes -Wmissing-prototypes -I ../../sonLib/lib -I ../inc -I ../external  -O3 src/mafExtractor.c ../lib/common.o ../lib/sharedMaf.o ../external/CuTest.a src/mafExtractorAPI.o src/buildVersion.o -o ../bin/mafExtractor.tmp
../lib/sharedMaf.o: In function `maf_mafBlock_print':
sharedMaf.c:(.text+0x2406): undefined reference to `log10'
sharedMaf.c:(.text+0x2435): undefined reference to `log10'
sharedMaf.c:(.text+0x2464): undefined reference to `log10'
collect2: error: ld returned 1 exit status
make: *** [../bin/mafExtractor] Error 1
```
I added `-lm` to the line compiling mafExtractor in the makefile for mafExtrator and that seemed to compile correctly.

I've tried adding -lm to every line with ${cxx}, and mine still is failing to compile, with the following errors (makefile attached):
mkdir -p ../bin/
gcc -std=c99 -Wno-unused-but-set-variable -lm -O3 -Wall -Werror --pedantic -funroll-loops -DNDEBUG -Wshadow -Wpointer-arith -Wstrict-prototypes -Wmissing-prototypes -I ../../sonLib/lib -I ../inc -I ../external  -O3 src/mafExtractor.c ../lib/common.o ../lib/sharedMaf.o ../external/CuTest.a src/mafExtractorAPI.o src/buildVersion.o -o ../bin/mafExtractor.tmp
../lib/sharedMaf.o: In function `maf_mafBlock_print':
sharedMaf.c:(.text+0x24e0): undefined reference to `log10'
sharedMaf.c:(.text+0x251a): undefined reference to `log10'
sharedMaf.c:(.text+0x2554): undefined reference to `log10'
collect2: error: ld returned 1 exit status
Makefile:24: recipe for target '../bin/mafExtractor' failed
make: *** [../bin/mafExtractor] Error 1

[Makefile.txt](https://github.com/dentearl/mafTools/files/3234358/Makefile.txt)

--- a/mafExtractor/Makefile
+++ b/mafExtractor/Makefile
@@ -22,12 +22,12 @@ src/buildVersion.c: ${sources} ${dependencies}
 
 ${bin}/mafExtractor: src/mafExtractor.c ${dependencies} ${API}
 	mkdir -p $(dir $@)
-	${cxx} ${cflags} -O3 $< ${API} -o $@.tmp
+	${cxx} ${cflags} -O3 $< ${API} -o $@.tmp -lm
 	mv $@.tmp $@
 
 test/mafExtractor: src/mafExtractor.c ${dependencies} ${testAPI}
 	mkdir -p $(dir $@)
-	${cxx} ${cflags} -g -O0 $< ${testAPI} -o $@.tmp
+	${cxx} ${cflags} -g -O0 $< ${testAPI} -o $@.tmp -lm
 	mv $@.tmp $@
 
 %.o: %.c %.h

